packages/gasket-react-intl/src/index.d.ts

```diff
@@ -38,7 +38,7 @@ export type LocaleRequiredWrapper = (props: {
  */
 export function withLocaleRequired<Props>(
   /** Path containing locale files */
-  localePathPart?: LocalePathPartOrThunk,
+  localePathPart?: LocalePathPartOrThunk | LocalePathPartOrThunk[],
```

Comment:
If you're interested, there's a `MaybeMultiple` (or perhaps it was called `MaybeArray`) type that you can use.